### DISCUSSION TITLE
KOG Meetings

### DISCUSSION THREAD

**Discussion by slysnivy** [shoc71/dont-stop-now#318] [Ideas]
# We will post the main points of every meeting here. We should aim to discuss these things:
1. **What important topics** are there to discuss (new ideas/features, game direction)
2. **How** will this **change/topic affect** the game and influence it's direction (code, visuals/graphics, gameplay)
3. What we should **focus** on for that **week**

**Comment by slysnivy** [comment]
Before the meeting on September 12th 2023, I want to discuss and work on @shoc71:
- The Main Story
- What existing levels do we need to keep or remove
- What we need to add to the game to complement the main story
- Level 0

**Reply by slysnivy** [reply]
Do you have anything that you want to discuss?

**Reply by shoc71** [reply]
Nope, this seems like everything! Everything seems good!

**Reply by slysnivy** [reply]
@shoc71 Remember to post your notes

**Comment by slysnivy** [comment]
For today's meeting (September 18th) I'd suggest we discuss these things:
- Hub zone designs, character designs/personalities, level theming
- level zero (if needed)
- And lastly, anything you don't understand or questions about the code

If there's anything else you'd like to discuss lmk

**Comment by slysnivy** [comment]
For today's meeting (September 26th 2023), there's nothing I'd like to discuss. Therefore, we can have sort of a free discussion.

**Comment by slysnivy** [comment]
I don't think we had meetings on October 3rd, 10th and 17th

**Comment by slysnivy** [comment]
Meeting October 24th was for resolving the issue of incorrect file versions on github due to faulty pull requests and for revising next steps on the game after a long break.

**Comment by slysnivy** [comment]
Meeting October 31st was mainly for building pause screen

**Comment by slysnivy** [comment]
For today's meeting (November 7th), we are going to finish HubZones and discuss future features such as NPC's in their design, actions and dialogue, and the Animate class to add in needed elements.

**Comment by slysnivy** [comment]
@shoc71 For November 14th, I want to discuss and work on specific details on HubZones. More specifically:
- Designing HubZones to include important features such as HubSelect, LevelSelect, Options, and NPC locations
- Changing the game flow that we have and make the player start in the Hubzone and then choose Hubselect/Levelselect
- Plan what we should be working on until next meeting or until the end of the year (vision)

If there's anything else you'd like to discuss feel free to add it in a reply.

**Comment by shoc71** [comment]
For November 21st, here are a few things that I would like to discuss about;
- Being able to transition from one hubzone to the next hubzone
- switching up the order of the pause functions and main menu
- play specific music for specific hubzones

If there's anything else you'd like to discuss feel free to add on to this list.

**Reply by slysnivy** [reply]
- Building onto the existing story
- The editor

**Comment by shoc71** [comment]
For 5th December meeting,

- gameplay testing (making sure hubzones work and are in the right order)
- getting specific music tracks to play in specific hubzones only
- making sure there is a stars(points)-system in play for passing levels and stuff
- correcting the formatting for the time displayed, all across the game

I believe these are simple issues that can be solved in today's meeting time. If there are any additional things/issues that need to be addressed - leave them in the reply section below and it shall act as the guide for today's meeting.

**Reply by shoc71** [reply]
- next button for playing a new song

**Comment by slysnivy** [comment]
Here are the things I want to discuss for December 18th's meeting:
- Post-reviewing the levels
- Ensuring we have a solid story
- Changing how levels are being made to have them based off of a story
- Development timeline for next year
- If we have time, choosing suitable sound effects for our game
